arch/risc-v/src/esp32c3/Kconfig

```diff
@@ -261,6 +262,29 @@ config ESP32C3_SPIFLASH
 	select MTD_BYTE_WRITE
 	select MTD_PARTITION
 
+config ESP32C3_SPI_LCD
+	bool "Generic SPI LCD Module"
+	default n
+	select ESP32C3_SPI2
+	select SPI_CMDDATA
+	select ESP32C3_SPI_SWCS
+	select SPI_SWAPBYTES
```

Comment:
I believe there is no need to create this `Generic SPI LCD Module` entry in Kconfig.
I understand that this is just a `shortcut` to selecting each individual config.
And by the way, I think that defaulting to `ESP32C3_SPI_SWCS` breaks the "generic" qualifier for this `ESP32C3_SPI_LCD` config, since one may want to not use the CS via software.

Reply:
One suggestion in case you remove created this config as a shortcut:
you can create a defconfig under `nuttx/boards/risc-v/esp32c3/esp32c3-devkit/configs`